src/components/cursor.js

```diff
@@ -33,60 +38,118 @@ module.exports.Component = registerComponent('cursor', {
   dependencies: ['raycaster'],
 
   schema: {
+    downEvents: {default: []},
```

Comment:
can this be conditionally set to be `trackpaddown` and `triggerdown`? otherwise, that's quite a bit of manual work for devs